test/colors-spec.js

```diff
@@ -43,10 +45,15 @@ describe('colors', function () {
     errors = []
     consoleError = console.error
     console.error = (...args) => errors.push(args)
+
+    infos = []
+    consoleInfo = console.info
+    console.info = (...args) => infos.push(args)
```

Comment:
This may be overkill for this PR but there is a common module that people use to follow this kind of pattern called [`sinon`](https://npmjs.com/sinon)

```js
const sinon = require('sinon')

sinon.stub(console, 'info') // Stubs the `info` method of `console`
console.info('foo') // Does not log
expect(console.info.callCount).to.equal(1) // You can do assertions on calls
expect(console.info.firstCall.args).to.deep.equal(['foo'])

console.info.restore() // Restores original implementation
```